samples/language/Result.qs

```diff
@@ -0,0 +1,30 @@
+/// # Sample
+/// Result
+///
+/// # Description
+/// Represents the result of a projective measurement onto the eigenspaces
+/// of a quantum operator with eigenvalues ±1. Possible values are `Zero` or `One`.
+namespace MyQuantumApp {
+
+    // Open the `Microsoft.Quantum.Measurement` namespace to import `MResetZ`.
+    open Microsoft.Quantum.Measurement;
```

Comment:
Since the focus of this sample is the `Result` type, I would use `M` instead of `MResetZ` and do an explicit `Reset` on the qubit.